src/job/productionJobHandler.ts

```diff
@@ -100,7 +100,8 @@ export class ProductionJobHandler extends JobHandler {
       const updatedURLsArray = stdoutJSON.urls;
       // purgeCache purges the now stale content and requests the URLs to warm the cache for our users
       await this.logger.save(this.currJob._id, JSON.stringify(updatedURLsArray));
-      const id = await this._cdnConnector.purge(this.currJob._id, updatedURLsArray);
+      console.log('this.currJob.payload.prefix : ' + this.currJob.payload.prefix);
```

Comment:
If we want to retain this log statement, we might want to log out a little more context here, to help anyone reading the log outputs in CloudWatch.